_posts/2025-04-24-Announcing-Data-Prepper-2.11.md

```diff
@@ -0,0 +1,276 @@
+through Data Prepper's new Confluence source plugin. This integration enables organizations to create a centralized, searchable repository of their collective knowledge,
+fostering improved information discovery and team productivity. By synchronizing Confluence content and continuously monitoring for updates,
+the plugin ensures that your OpenSearch index remains an up-to-date, comprehensive reflection of your organization's shared knowledge base.
+The integration offers flexible filtering options, allowing you to selectively import content from specific spaces or page types, tailoring the synchronized content to your organization's needs.
+The plugin supports both basic API key and OAuth2 authentication methods, with the added option of securely managing credentials through AWS Secrets.
+Furthermore, the plugin's automatic token renewal feature guarantees uninterrupted access and seamless operation.
+Built on Atlassian's Confluence [api-version](https://developer.atlassian.com/cloud/confluence/rest/v1/intro/#auth),
+this integration empowers teams to leverage OpenSearch's advanced search capabilities across their Confluence content,
+dramatically enhancing information accessibility and utilization within the organization. Here's how to get started:
+
+```
+version: "2"
+extension:
+    aws:
+      secrets:
+        confluence-account-credentials:
+          secret_id: <<secret-arn>>
+          region: <<secrets-region>>
+          sts_role_arn: <<role-to-access-secret>>
+confluence-pipeline:
+  source:
+    confluence:
+      hosts: ["<<Atlassian-host-url>>"]
+      authentication: 
+        basic:
+          username: ${{aws_secrets:confluence-account-credentials:confluenceId}}
+          password: ${{aws_secrets:confluence-account-credentials:confluenceCredential}}
+      filter:
+        space:
+          key:
+            include:
+              - "<<space-key>>"
+            exclude:
+               - "<<space-key>>"
+        page_type:
+          include: 
+            - "page"
+            - "blogpost"
+            - "comment"
+          exclude:
+            - "attachment"
+  sink:
+    - opensearch:
+
+```
+
+
+## Amazon Aurora/RDS as a source
+
+[Amazon Aurora](https://aws.amazon.com/rds/aurora/) and [Amazon RDS](https://aws.amazon.com/rds/) are fully managed relational database services that make it easier to set up, operate, and scale a relational database in the AWS Cloud.
+For those who want to take advantage of advanced search capabilities like full-text and vector search on the transactional data in Amazon Aurora/RDS, you can now use Data Prepper to synchronize data from Aurora and RDS to OpenSearch.
+
+Data Prepper’s new `rds` source first exports existing data from Amazon Aurora/RDS tables to OpenSearch indices, then streams incremental changes from those tables to keep the data consistent between the relational database and OpenSearch.
```

Comment:
```suggestion
Data Prepper's new `rds` source first exports existing data from Aurora/Amazon RDS tables to OpenSearch indexes and then streams incremental changes from those tables in order to maintain data consistency between the relational database and OpenSearch.
```

Reply:
Should it be Amazon Aurora / Amazon RDS?
```suggestion
Data Prepper’s new `rds` source first exports existing data from Amazon Aurora/Amazon RDS tables to OpenSearch indices, then streams incremental changes from those tables to keep the data consistent between the relational database and OpenSearch.
```

Reply:
See above